cmake/Dependencies.cmake

```diff
@@ -11,9 +11,6 @@ set(CMAKE_SKIP_BUILD_RPATH  FALSE)
 # Don't use the install-rpath during the build phase
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 set(CMAKE_INSTALL_RPATH "${_rpath_portable_origin}")
```

Comment:
Why not delete this line as well?

Reply:
Caution mainly. I think an `$ORIGIN` directive in `RPATH` should be harmless and may actually help in certain cases.